src/v/coproc/pacemaker.cc

```diff
@@ -68,7 +68,9 @@ ss::future<> pacemaker::start() {
     co_await ss::recursive_touch_directory(offsets_snapshot_path().string());
     auto ncc = co_await recover_offsets(_offs.snap, _shared_res.api.log_mgr());
     _ntps = std::move(ncc);
-    _offs.timer.arm(_offs.duration);
+    if (!_offs.timer.armed()) {
```

Comment:
does this imply that start() is called more than once? that's a bit of a dark pattern and needs to be documented if it is the case.